frontend/src/app/components/HeroSection/page.tsx

```diff
@@ -0,0 +1,37 @@
+"use client"
+import Image from 'next/image';
+import { Archivo_Black } from "next/font/google";
+import SignButton from '../ui/SignButton';
+
+const HeroSection = () => {
+    return ( 
+        <>
+        <section className="h-full w-full relative bg-[#f9f4ee]">
+            <div className="h-screen w-full flex justify-around items-center">
+            
+                    <Image src='/HeroRight.png' width={600} height={600} alt="hero" className="object-contain absolute right-0 -top-20" />
+                    <Image src='/HeroLeft.png' width={200} height={200} alt="hero" className="object-contain absolute left-0 top-40" />
+                    <Image src='/DJANGO.svg' width={816} height={164} alt="logo" className="absolute left-40 top-32"/>
+                    <Image src='/INDIA.svg' width={532} height={159} alt="logo" className="absolute right-60 top-80"/>
```

Comment:
One minor change, add `unoptimized` prop to all Image components where file is know to be an svg.

Check: https://nextjs.org/docs/pages/api-reference/components/image#dangerouslyallowsvg

Same thing in `frontend/src/app/components/Navbar/page.tsx` also on `line 11`